enclave/enclave-api/src/main/java/com/quorum/tessera/enclave/PrivacyGroupUtil.java

```diff
@@ -0,0 +1,118 @@
+package com.quorum.tessera.enclave;
+
+import com.quorum.tessera.encryption.PublicKey;
+import org.apache.tuweni.bytes.Bytes;
+import org.apache.tuweni.rlp.RLP;
+import org.bouncycastle.jcajce.provider.digest.Keccak;
+
+import java.nio.ByteBuffer;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public interface PrivacyGroupUtil extends BinaryEncoder {
+
+    byte[] BYTES = Bytes.fromHexString("5375ba871e5c3d0f1d055b5da0ac02ea035bed38").toArrayUnsafe();
```

Comment:
what does this hex string signify? Can it have a descriptive name?

Reply:
This seems to be what Orion uses to generate the id for querying purpose - https://github.com/ConsenSys/orion/blob/7b66adb22abb11d68360a4b055abcf811f5e685b/src/main/java/net/consensys/orion/storage/QueryPrivacyGroupStorage.java#L35

and it was copied over as Tessera needs to result the same value when generating lookup id

Reply:
can we add a comment here saying "this matches the same value in Orion"?

Reply:
Yup sure. Comment added